src/components/ParadeHeader.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+import React from "react";
+import { View, StyleSheet } from "react-native";
+
+import Text, { scaleWithFont } from "./Text";
+import { lightNavyBlueColor } from "../constants/colors";
+import text from "../constants/text";
+
+const ParadeHeader = () => (
+  <View style={styles.wrapper} testID="page-heading-parade">
```

Comment:
I think we should be able to use our common `Header` component here. Then we don't need to duplicate the styles.

Reply:
Sounds good to me, I'll do that